examples/hydropower_example.json

```diff
@@ -98,7 +98,7 @@
             "storage_node": "reservoir1",
             "level": 35.0,
```

Comment:
I had to change `level` to `turbine level` and replace `storage_node` with `water_elevation_parameter`, pointing to the interpolated volume parameter